### DISCUSSION TITLE
[A-L] (16 Gennaio 2023) Esercizio 6

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#369] [Esami A-L]
Consideriamo la seguente dimostrazione per Induzione Forte.

- Tesi:
    
    Per ogni $\Large n \geq 0,\ 12 \times n = 0$.

- Caso Base:

    Per $\Large n = 0$ abbiamo $\Large 12 \times 0 = 0$.

- Passo Induttivo:

    Sia $\Large n \geq 0$ e assumiamo che la tesi valga per ogni intero $\Large k$ tale che $\Large 0 \leq k \leq n$.

    Dimostriamo che vale per $\Large n + 1$. Scriviamo  $\Large n + 1$ come  $\Large a + b$ per qualche $\Large a, b$ tali che $\Large 0 \leq a,\ b \leq n$.

    Per ipotesi induttiva vale $\Large 12 \times a = 0$ e $\Large 12 \times b = 0$. 

    Dunque $\Large 12 \times (n + 1) = 12 \times (a + b) = (12 \times a) + (12 \times b) = 0 + 0 = 0$.

Trovare l’errore o gli errori nell’argomento sopra esposto (NB: non è sufficiente dire che la tesi dimostrata è falsa - questo è ovvio).
> si, ha veramente scritto quel nota bene nella traccia

Indicare esplicitamente se il Caso Base e il Passo Induttivo sono corretti.

**Comment by MATTEOHH** [comment]
Caso Base: corretto
Passo induttivo: sbagliato 
Nel caso n = 0 avremmo, come scritto nella traccia, n + 1 = a + b = 0 + 1 = 1, questo contraddice 0 ≤ a, b ≤ n perché non esiste nessun valore di a,b tale che 0 ≤ a, b ≤ n (sarebbe come scrivere 0 ≤ 1 ≤ 0, che è impossibile).
